gulpfile.js

```diff
@@ -0,0 +1,37 @@
+var gulp = require('gulp');
```

Comment:
I suggest we use `const` instead of `var`. Formatting of the file is also not ideal. Some tasks have blank line in between, others do not.